client/my-sites/people/invite-people/index.jsx

```diff
@@ -728,10 +768,11 @@ const connectComponent = connect(
 	( dispatch ) => ( {
 		...bindActionCreators(
```

Comment:
Optional: we can totally get rid of this one - binding actions to `dispatch` is now automatic when we return an object with all unbound action creators. Wanna remove this thorn out of my eye? 😉